src/github.com/getlantern/flashlight/config/fetcher.go

```diff
@@ -49,8 +48,8 @@ type UserConfig interface {
 
 // NewFetcher creates a new configuration fetcher with the specified
 // interface for obtaining the user ID and token if those are populated.
-func NewFetcher(conf UserConfig, httpFetcher util.HTTPFetcher) Fetcher {
-	return &fetcher{lastCloudConfigETag: map[string]string{}, user: conf, httpFetcher: httpFetcher}
+func NewFetcher(conf UserConfig, httpClient *http.Client) Fetcher {
```

Comment:
Also, http.Client has the advantage of being defined in the standard library, so more likely to just be something that we can pass around.  If we go to an interface, I would use http.RoundTripper which is actually identical to our HTTPFetcher.